### DISCUSSION TITLE
Layout of the authentication page

### DISCUSSION THREAD

**Discussion by johannadm** [netwerk-digitaal-erfgoed/solid-crs#69] [Authentication]
Hi all,

In this discussion I'd like to hear your opinion on the layout and design of the authentication page. The mockups can be seen in Figma, but to be clear I also show them below. 

1. Authentication page when no WebID is entered yet
![afbeelding](https://user-images.githubusercontent.com/80322390/116537991-409ee900-a8e7-11eb-95ca-c2bfeb4227fd.png)

2. When a non valid WebID is entered, a form validation result shows up
![afbeelding](https://user-images.githubusercontent.com/80322390/116541415-b9a03f80-a8eb-11eb-9cfe-66636be8aae1.png)

3. When the authentication at the OIDC provider did not succeed (e.g. the user did not finish the authentication), the user is redirected to the authentication page of the CBS and an alert shows up
![afbeelding](https://user-images.githubusercontent.com/80322390/116539140-bbb4cf00-a8e8-11eb-91c1-0fb002d8c16b.png)

**Comment by EnnoMeijers** [comment]
From the design perspective I am ok with it but I am missing a pointer to an instruction on how to acquire a WebID, so something like 'nog geen WebID'...

**Reply by johannadm** [reply]
That's missing indeed. I adjusted it in the mockup like this: 
![afbeelding](https://user-images.githubusercontent.com/80322390/116552305-2d951480-a8f9-11eb-919e-b8c6f4b22081.png)
